webhook_xblock/static/html/webhook_xblock_edit.html

```diff
@@ -0,0 +1,69 @@
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+  <ul class="list-input settings-list">
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="webhook_url">URL</label>
+        <input class="input setting-input" style="width: 550px;" name="webhook_url" id="webhook_url" value="{webhook_url}" type="url" required/>
```

Comment:
Why don't you use the .css file instead of inline style?

Reply:
@mariajgrimaldi done!